<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test"
           tests="3"
           failures="1"
           errors="0"
           skipped="2"
           time="0.001">
  <properties />
            <testcase name="setUp" time="0.000" classname="com.sysco.test.WorkFlow">
                <failure type="org.openqa.selenium.WebDriverException"
                  message="An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 22.32 seconds
Build info: version: &apos;2.53.1&apos;, revision: &apos;a36b8b1&apos;, time: &apos;2016-06-30 17:37:03&apos;
System info: host: &apos;Raghavendras-MacBook-Air.local&apos;, ip: &apos;10.242.147.21&apos;, os.name: &apos;Mac OS X&apos;, os.arch: &apos;x86_64&apos;, os.version: &apos;10.11.6&apos;, java.version: &apos;1.8.0_91&apos;
Driver info: io.appium.java_client.android.AndroidDriver"
              >
<![CDATA[org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device. (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 22.32 seconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:37:03'
System info: host: 'Raghavendras-MacBook-Air.local', ip: '10.242.147.21', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.6', java.version: '1.8.0_91'
Driver info: io.appium.java_client.android.AndroidDriver
  sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
  sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
  sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
  java.lang.reflect.Constructor.newInstance(Constructor.java:423)
  org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
  org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
  org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
  io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:51)
  io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
  io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
  org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:249)
  org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
  org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:144)
  io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:47)
  io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:114)
  io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:132)
  io.appium.java_client.android.AndroidDriver.<init>(AndroidDriver.java:97)
  com.sysco.test.WorkFlow.setUp(WorkFlow.java:64)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
  org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:585)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
      </failure>

          </testcase>
  
            <testcase name="clear" time="0.0" classname="com.sysco.test.WorkFlow">
          <skipped />
    </testcase>
            <testcase name="SI_WF1_OG_DefaultLocation_DefaultCategory" time="0.0" classname="com.sysco.test.WorkFlow">
          <skipped />
    </testcase>
  
  
</testsuite>
